build.sbt

```diff
@@ -53,22 +53,6 @@ ThisBuild / githubWorkflowAddedJobs ++= Seq(
   )
 )
 
-ThisBuild / githubWorkflowEnv ++= Map(
```

Comment:
Why is this no longer needed?

Reply:
I convinced Daniel to take all the boilerplate needed to publish to Sonatype and put it in sbt-spiewak. We get all the removed config via the new `SonatypeCiRelease` plugin enabled on the root project. Documented here: https://github.com/djspiewak/sbt-spiewak#ci-release